native/objcexport-header-generator/src/org/jetbrains/kotlin/backend/konan/objcexport/parameterNames.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2010-2025 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.backend.konan.objcexport
+
+import org.jetbrains.kotlin.tooling.core.MutableExtras
+import org.jetbrains.kotlin.tooling.core.extrasKeyOf
+
+val originParameterNamesKey = extrasKeyOf<List<String>>()
```

Comment:
A generic type as 'List<String>' would certainly require to give the key a name as well. 
I suggest `"org.jetbrains.kotlin.backend.konan.objcexport.originParameterNamesKey"`